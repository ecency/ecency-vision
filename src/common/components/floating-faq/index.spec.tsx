import React from "react";
import FloatingFAQ from "./index";
import renderer from "react-test-renderer";
import { BrowserRouter } from "react-router-dom";

it("(1) Default render", () => {
  const props = {};
  const component = renderer.create(
    <BrowserRouter>
      <FloatingFAQ {...props} />
    </BrowserRouter>
  );
  expect(component.toJSON()).toMatchSnapshot();
});
