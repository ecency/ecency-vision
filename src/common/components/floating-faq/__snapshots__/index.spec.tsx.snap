// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(1) Default render 1`] = `
<div
  className="floating-faq-button"
>
  
</div>
`;
